azure-pipelines/templates/wheel-version.yml

```diff
@@ -0,0 +1,28 @@
+# Template to edit dev build versions
+steps:
+  - bash: |
+      set -e
+      echo "BUILD_BUILDID: $BUILD_BUILDID"
+      VERSION_SUFIX="$(echo $BUILD_BUILDID | sed -E 's/[- ]/_/g')"
+      echo "VERSION_SUFIX: $VERSION_SUFIX"
+      VERSION_FILE="$BUILD_SOURCESDIRECTORY/tensorflow/tools/pip_package/setup.py"
+      echo "Version file: $VERSION_FILE"
+
+      if [[ ! -f "$VERSION_FILE" ]]; then
+        echo File $VERSION_FILE does not exist.
+        exit 1
+      fi
+
+      if [[ "$BUILD_SOURCEBRANCH" == *"master"* ]] || [[ "$BUILD_SOURCEBRANCH" == *"releases/"* ]]; then
+        VERSION_SUFIX=".${VERSION_SUFIX}"
+      echo "Release generation detected. The VERSION_SUFIX will be directly appended."
```

Comment:
NIT: will the indentation cause any issue? - I'm not super good at bash+azure-pipelines-yml but I can of remember having issues with it.

Reply:
didn't seem in my debugging. Shell does not care about indentation. However, it is a good point for style. i'll fix it.